tools/lib/glbToI3dm.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var Cesium = require('cesium');
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+var DeveloperError = Cesium.DeveloperError;
+
+module.exports = glbToI3dm;
+
+/**
+ * Generates a new Buffer representing a i3dm asset.
+ *
+ * @param {Buffer} glbBuffer A buffer containing a binary glTF asset.
+ * @param {Buffer} [featureTableJSONBuffer] A buffer containing the batch table to use for the i3dm asset.
+ * @param {Buffer} [featureTableBinaryBuffer] A buffer containing the accompanying binary batch table section for the i3dm asset.
+ * @param {Buffer} [batchTableJSONBuffer] A buffer containing the batch table to use for the i3dm asset.
+ * @param {Buffer} [batchTableBinaryBuffer] A buffer containing the accompanying binary batch table section for the i3dm asset.
+ * @returns {Buffer} Buffer representing the i3dm asset.
+ */
+function glbToI3dm(glbBuffer, featureTableJSONBuffer, featureTableBinaryBuffer, batchTableJSONBuffer, batchTableBinaryBuffer) {
+    if (!defined(glbBuffer)) {
+        throw new DeveloperError('glbBuffer is not defined.');
+    }
+
+    if (!defined(featureTableJSONBuffer)) {
+        throw new DeveloperError('featureTableJSONBuffer is not defined.');
+    }
+
+    if (!defined(featureTableBinaryBuffer)) {
+        throw new DeveloperError('featureTableBinaryBuffer is not defined.');
+    }
```

Comment:
I'm fine with removing these two defined checks. I wouldn't consider it an error if the feature table is missing - it will produce invalid i3dms, but that should be up to the code that calls this.